src/cls/IPM/Utils/Module.cls

```diff
@@ -1206,8 +1206,9 @@ ClassMethod LoadNewModule(pDirectory As %String, ByRef pParams, pRepository As %
 		}
 		Merge $$$ZPMHandledModules($Namespace) = pParams("Multicompile","ModuleContext")
 
-    $$$ThrowOnError(##class(%IPM.Storage.Module).CheckSystemRequirements(tModuleName))
-  
+		$$$ThrowOnError(##class(%IPM.Storage.Module).CheckSystemRequirements(tModuleName))
+		Set pParams("RootDirectory") = pDirectory
```

Comment:
What is the purpose of setting RootDirectory here instead of below? That seems to be the only functional change so if its not needed, changes to this file can be reverted as well